Automattic/newspack-content-converter
# Pipeline

This PR restructures the build pipeline to closely resemble `[newspack-plugin](https://github.com/Automattic/newspack-plugin)` and `[newspack-blocks](https://github.com/Automattic/newspack-blocks)`. I tried to keep each commit to a single change, in the hope that reading through the commit messages in order will give a clear view of what's happening here and why. 

Testing instructions:

- Check out `update/pipeline`
- `npm ci && npm run clean && npm run build:webpack`
- Verify that the app works exactly as before.
